star-rx72n-firmware/lib/rx_motor/src/rx_motor.c

```
@@ -185,6 +241,9 @@ rx_err_t rx_motor_deinit(rx_motor_handle_t* handle)
 
 rx_err_t rx_motor_set_duty(rx_motor_handle_t* handle, float duty)
 {
+  float    speed_pwm = 0.0f;
```

Clang (14.0.6)

[warning] floating point literal has suffix 'f', which is not uppercase

(readability-uppercase-literal-suffix)